src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1065,11 +1065,19 @@ def config_diagnostics(resource_group_name, name, level=None,
         application_logs = ApplicationLogsConfig(fs_log)
 
     http_logs = None
-    if web_server_logging is not None:
-        enabled = web_server_logging
-        # 100 mb max log size, retenting last 3 days. Yes we hard code it, portal does too
-        fs_server_log = FileSystemHttpLogsConfig(100, 3, enabled)
-        http_logs = HttpLogsConfig(fs_server_log)
+    logs_enabled = (web_server_logging if web_server_logging is not None
+                    else docker_container_logging)
+
+    # 100 mb max log size, retenting last 3 days. Yes we hard code it, portal does too
```

Comment:
You can guess my move now.

Reply:
Why isn't this a default that can be configured?

Reply:
I think this is fine?